LightBridge deployed to: 0x3f7Da9C51138E0475aA26E80677d27A568cFD6b9
Proxy__LightBridge deployed to: 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Initialized proxy: [object Object]
Teleportation initialized: [object Object]
Added route for 11155111 chain, and token: 0x0000000000000000000000000000000000000000, receipt: [object Object]
Added route for 11155111 chain, and token: 0x4200000000000000000000000000000000000023, receipt: [object Object]
Network iteration done
Using previous deployment of LightBridge contract on 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Could not initialize contracts for already deployed contracts: {}
Using previous deployment of LightBridge contract on 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Could not initialize contracts for already deployed contracts: {}
Could not add route for 11155111 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0x48baCac867ef377fac62186A79e8381665cfae6B","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155111 chain, and token: 0x4200000000000000000000000000000000000023: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0x48baCac867ef377fac62186A79e8381665cfae6B","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000042000000000000000000000000000000000000230000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000001158e460913d0000000000000000000000000000000000000000000000000043c33c1937564800000000000000000000000000000000000000000000000000878678326eac9000000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155420 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0x48baCac867ef377fac62186A79e8381665cfae6B","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa37dc000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 421614 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0x48baCac867ef377fac62186A79e8381665cfae6B","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000066eee000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Network iteration done
Using previous deployment of LightBridge contract on 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Could not initialize contracts for already deployed contracts: {}
Could not add route for 11155111 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155111 chain, and token: 0x4200000000000000000000000000000000000023: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000042000000000000000000000000000000000000230000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000001158e460913d0000000000000000000000000000000000000000000000000043c33c1937564800000000000000000000000000000000000000000000000000878678326eac9000000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155420 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa37dc000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 421614 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000066eee000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Network iteration done
Using previous deployment of LightBridge contract on 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Could not initialize contracts for already deployed contracts: {}
Could not add route for 11155111 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155111 chain, and token: 0x4200000000000000000000000000000000000023: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000042000000000000000000000000000000000000230000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000001158e460913d0000000000000000000000000000000000000000000000000043c33c1937564800000000000000000000000000000000000000000000000000878678326eac9000000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 11155420 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000aa37dc000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Could not add route for 421614 chain, and token: 0x0000000000000000000000000000000000000000: {"reason":"execution reverted: Caller is not the owner","code":"UNPREDICTABLE_GAS_LIMIT","method":"estimateGas","transaction":{"from":"0xc5E61755eeeD4f8A37CBb5386F9397c33707c4b2","to":"0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801","data":"0x5915dc8a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000066eee000000000000000000000000000000000000000000000000002386f26fc100000000000000000000000000000000000000000000000000001bc16d674ec8000000000000000000000000000000000000000000000000000053444835ec580000","accessList":null},"error":{"name":"ProviderError","_stack":"ProviderError: execution reverted: Caller is not the owner\n    at HttpProvider.request (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\hardhat@2.22.1_ts-node@10.9.2_typescript@4.9.5\\node_modules\\hardhat\\src\\internal\\core\\providers\\http.ts:90:21)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async EthersProviderWrapper.send (C:\\Users\\wsdt\\Desktop\\GitRepositories\\Boba\\light-bridge\\node_modules\\.pnpm\\@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.1\\node_modules\\@nomiclabs\\hardhat-ethers\\src\\internal\\ethers-provider-wrapper.ts:13:20)","code":3,"_isProviderError":true,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001743616c6c6572206973206e6f7420746865206f776e6572000000000000000000"}}
Network iteration done
Using previous deployment of LightBridge contract on 0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801
Could not initialize contracts for already deployed contracts: {}
Route for 11155111 chain, and token: 0x0000000000000000000000000000000000000000 not added again, as already supported.
Route for 11155111 chain, and token: 0x4200000000000000000000000000000000000023 not added again, as already supported.
Added route for 11155420 chain, and token: 0x0000000000000000000000000000000000000000, receipt: [object Object]
Added route for 421614 chain, and token: 0x0000000000000000000000000000000000000000, receipt: [object Object]
Network iteration done
